docs/EN_US/ContainerizedHPCC/ContainerizedMods/RemoteAccess.xml

```diff
@@ -0,0 +1,347 @@
+
+          <para>All those prefixed with <emphasis
+          role="bold">client-remote-</emphasis> and <emphasis
+          role="bold">client-remote-dfs-</emphasis> are the secrets to use for
+          the client(s) configured earlier.</para>
+
+          <para>Extract the certificate and key pairs and save to files with
+          these commands:</para>
+
+          <para><programlisting>kubectl get secret client-remote-dfs-dfs-env2-tls -o json | jq -r '.data."tls.crt"'| base64 -d &gt; env2tls.crt 
+kubectl get secret client-remote-dfs-dfs-env2-tls -o json | jq -r '.data."tls.key"'| base64 -d &gt; env2tls.key 
+kubectl get secret client-remote-dfs-dfs-env2-tls -o json | jq -r '.data."ca.crt"' | base64 -d &gt; env2ca.crt </programlisting></para>
+
+          <para><emphasis role="bold">Keep a record of the files created here,
+          they are needed when configuring the client
+          environment.</emphasis></para>
+        </listitem>
+
+        <listitem>
+          <para>Get the DFS service external endpoint:</para>
+
+          <para><programlisting>kubectl get svc</programlisting></para>
+
+          <para>Example output:</para>
+
+          <para><programlisting>NAME                 TYPE          CLUSTER-IP    EXTERNAL-IP    PORT(S)         AGE 
+cert-manager         ClusterIP     10.0.1.53     &lt;none&gt;         9402/TCP        24h 
+cert-manager-webhook ClusterIP     10.0.13.136   &lt;none&gt;         443/TCP         24h 
+dfs                  LoadBalancer  10.0.108.249  <emphasis role="red">192.168.0.36   8520</emphasis>:32469/TCP  23h </programlisting></para>
+
+          <para><emphasis role="bold">Keep a record of the external dfs
+          service endpoint, in this example, it’s
+          192.168.0.36:8520.</emphasis></para>
+        </listitem>
+      </orderedlist></para>
+  </sect3>
+
+  <sect3>
+    <title>Setting up the second environment (env2)</title>
+
+    <para><orderedlist>
+        <listitem>
+          <para>Create <emphasis role="bold">env2</emphasis> namespace (if it
+          does not already exist).</para>
+
+          <para><programlisting>kubectl create ns env2</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Switch to the <emphasis role="bold">env2</emphasis>
+          namespace.</para>
+
+          <para><programlisting>kubectl config set-context --current --namespace=env2</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Create a K8s secret with the remote client certificate and key
+          pair that you saved when setting up env1:</para>
+
+          <para><variablelist>
+              <varlistentry>
+                <term>Note:</term>
+
+                <listitem>
+                  <para>The following example command should be entered on one
+                  line without the "\" connectors. The connectors are only
+                  included here so the code will fit within page
+                  margins.</para>
```

Comment:
see previous comment re. it works with backslashes in Linux shells, but it is prob. best to remove this command and change the command line to a single line to ensure it works in all use cases